test/test_stla.py

```diff
@@ -0,0 +1,231 @@
+import math
```

Comment:
Nit: can we call this file `test_levy_area.py`?

Reply:
Yes, I agree.